tuning_the_ecl_ekf.md

```diff
@@ -122,6 +122,10 @@ The .ulog format data can be parsed in python by using the [PX4 pyulog library](
 
 Most of the EKF data is found in the [ekf2\_innovations](https://github.com/PX4/Firmware/blob/master/msg/ekf2_innovations.msg) and [estimator\_status](https://github.com/PX4/Firmware/blob/master/msg/estimator_status.msg) uORB messages that are logged to the .ulog file.
 
+A python script that automatically generates analysis plots and metadata can be found [here](https://github.com/priseborough/Firmware/blob/master/Tools/ecl_ekf/process_logdata_ekf.py). To use this script file, cd to the Tools/ecl_ekf directory and enter 'python process_logdata_ekf.py \<logfile path name'. This saves performance metadata in a csv file named \<input file name>.mdat.csv and plots in a pdf file named \<inputfilename>.pdf
```

Comment:
@priseborough I assume you wanted to specify the link to the PX4 repo instead of your private repo?